src/components/Dropdown/Dropdown.module.css

```diff
@@ -113,28 +111,30 @@
 /**
  * Wraps the check icon that indicates a selected option.
  * Required even if option is unselected to prevent the option content from shifting in position.
- * 1) Take up the same amount of space as an unselected option's padding-left.
- * 2) Position the contained icon. It's shifted slightly to the right from center so that the 
- *    checkmark's visual "center of mass" is centered.
  */
 .selected-icon__container {
   color: var(--eds-theme-color-icon-brand-primary);
-
-  flex-shrink: 0; /* 1 */
+  /* Take up the same amount of space as an unselected option's padding-left. */
+  flex-shrink: 0; 
   width: var(--eds-size-6);
-
-  padding-left: var(--eds-size-1); /* 2 */
+  /**
+    * Position the contained icon. It's shifted slightly to the right from center so that the 
+    * checkmark's visual "center of mass" is centered. 
+    */
+  padding-left: var(--eds-size-1); 
   display: inline-flex;
   align-items: center;
   justify-content: center;
 }
 
 /**
- * The content of the dropdown option
- * 1) Wrap overflowing content onto multiple lines. We're doing this instead of truncating so that
- *    users can still see all the content.
+ * The content of the dropdown option.
  */
 .option__content {
-  overflow-wrap: normal; /* 1 */
+  /**
+    * Wrap overflowing content onto multiple lines. We're doing this instead of truncating so that
+    * users can still see all the content. 
+    */
```

Comment:
```suggestion
   * Wrap overflowing content onto multiple lines. We're doing this instead of truncating so that
   * users can still see all the content. 
   */
```